docs/basics.md

```diff
@@ -396,6 +396,30 @@ Here is an example of backends and servers definition:
 - `backend2` will forward the traffic to two servers: `http://172.17.0.4:80"` with weight `1` and `http://172.17.0.5:80` with weight `2` using `drr` load-balancing strategy.
 - a circuit breaker is added on `backend1` using the expression `NetworkErrorRatio() > 0.5`: watch error ratio over 10 second sliding window
 
+## Custom Error pages
+Custom error pages can be returned, in lieu of the default, according frontend-configured ranges of HTTP Status codes.  In the example below, if a 503 status is returned from the frontend "website", the custom error page at http://2.3.4.5/500.html is returned with the actual status code set in the HTTP header.  Note, the 500.html page itself is not hosted on traefik, but some other infrastructure.  The configured status code ranges are inclusive; that is, in the below example, the 500.html page will be returned for status codes 500 through, and including, 599.  
+
+```toml
+[frontends]
+  [frontends.website]
+  backend = "website"
+  [errors]
+    [error.network]
+    status = ["500-599"]
+    backend = "error"
+    query = "/500.html"
```

Comment:
It seems I originally misunderstood this point.  I like it though and will extend to include the {status} query